File: Utilisateur.java
public abstract class Utilisateur{
    protected int idUtilisateur;
    protected String nom;
    protected String prenom;
    protected String passwordHash;
    protected String emailPro;
    protected boolean actif;
    
   public Utilisateur(int idUtilisateur, String nom, String prenom, String passwordHash, String emailPro){
        this.idUtilisateur = idUtilisateur;
        this.nom = nom;
        this.prenom = prenom;
        this.passwordHash = passwordHash;
        this.emailPro = emailPro;
        this.actif = false;
    }
}

File: Web/model/Utilisateur.php
<?php
public abstract class Utilisateur{
    protected idUtilisateur;
    protected nom;
    protected prenom;
    protected passwordHash;
    protected emailPro;
    protected actif;
    
   public Utilisateur(idUtilisateur, nom, prenom, passwordHash, emailPro){
        $this->idUtilisateur = idUtilisateur;
        $this->nom = nom;
        $this->prenom = prenom;
        $this->passwordHash = passwordHash;
        $this->emailPro = emailPro;
        $this->actif = false;
    }
}
?>
